src/fund/hub/FundRouterMixin.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity 0.6.4;
+
+import "../../registry/IRegistry.sol";
+import "./IHub.sol";
+
+/// @title FundRouterMixin Contract
+/// @author Melon Council DAO <security@meloncoucil.io>
+/// @notice A mixin that provides routing for a given fund
+abstract contract FundRouterMixin {
```

Comment:
why not make these functions `public` instead so that both plugin and component contracts can use.